saas/models.py

```diff
@@ -3527,7 +3527,7 @@ def valid_for(self, **kwargs):
         """
         Returns valid (i.e. fully opted-in) subscriptions.
         """
-        return self.filter(grant_key=None, request_key=None, **kwargs)
+        return self.filter(grant_key=None, **kwargs)
```

Comment:
By removing the `request_key-None` in the filter, it changes the definition of the function `valid_for` which then returns subscriptions which are not valid.